app/pages/lab/workflow-viewer/task-helpers.cjsx

```diff
@@ -0,0 +1,58 @@
+React = require 'react'
+TriggeredModalForm = require 'modal-form/triggered'
+tasks = require '../../../classifier/tasks'
+{Markdown} = require 'markdownz'
+
+# All classes in this file take in `workflow` and `taskKey` as props
+
+# Render the task name box
+TaskName = React.createClass
+  displayName: 'TaskName'
+
+  isEmptyStr: (str) ->
+    str.replace(/^\s+|\s+$/g, '').length == 0
+
+  render: ->
+    task = @props.wf[@props.taskKey]
+    task_text = tasks[task.type].getTaskText(task)
```

Comment:
Another note on code style. Javascript variable/function naming convention is camelCase. To keep things consistent with the rest of the codebase, we should try to stick with that.

(Take my code style comments with a grain of salt, as folks have different styles and to each their own. In this case, keeping things consistent with the rest of the app helps other devs jump in when necessary.)